### ISSUE TITLE
fatal error: cusparse.h: No such file or directory

### ISSUE THREAD

**Issue by JivanRoquet** [MVIG-SJTU/AlphaPose#522]
When following all the steps in INSTALL.md. At this particular step:

```
python setup.py build develop --user
```
After some time the following error happens:

```
In file included from detector/nms/src/nms_kernel.cu:3:0:
/usr/local/lib/python3.6/dist-packages/torch/include/ATen/cuda/CUDAContext.h:12:10: fatal error: cusparse.h: No such file or directory
 #include <cusparse.h>
          ^~~~~~~~~~~~
compilation terminated.
error: command '/usr/local/cuda/bin/nvcc' failed with exit status 1
```

For the record, this happens on a pre-built Paperspace Gradient environment with CUDA/cuDNN pre-installed. Re-installing CUDA does not solve anything.

Taking a copy of `cusparse.h` and then putting it in the correct directory only moves the problem to the next missing file, and so on and so forth.

Also, checking that Torch recognises Cuda, yes it does.
```
Python 3.6.9 (default, Nov  7 2019, 10:44:02) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
>>> torch.cuda.is_available()
True
>>> torch.version.cuda
'10.0'
```

**Comment by Fang-Haoshu** [comment]
Hi, I am not sure why this happens. We haven't met that before. Maybe it relates to the environment path? For example the cuda is installed in another place etc.

**Comment by JivanRoquet** [comment]
Thanks for the quick feedback. In hindsight it does look like an issue with my environment rather than an issue specific to AlphaPose. Closing as it doesn't look like there's anything wrong with AP there.

**Comment by HobJob** [comment]
Same issue here! Just followed the install read.me steps with conda.
Any suggestion?

**Comment by shekarneo** [comment]
same here also with conda
